docs/dictionary/command/play.lcdoc

```diff
@@ -11,6 +11,8 @@ Plays a movie or sound.
 
 Introduced: 1.0
 
+Deprecated: 8.1
```

Comment:
@livecodesam My bad - this line ^^^ should be removed, since the `play` command is not deprecated. It is just the `play videoClip ..` form that is deprecated.